_includes/footer.html

```diff
@@ -1,65 +1,106 @@
-                <p><small>Women Coding Community is a not-for-profit organisation.</small></p>
-                <p><small>© 2024 Women Coding Community</small></p>
-            </div>
-        </div>
+  <div class="container">
+    <div class="row justify-content-between">
+      <div class="col-12 col-md-8">
+        <img class="brand" src="/assets/images/logo.svg" />
+      </div>
+      <div class="col-12 col-md-4 social">
+        <h2 class="follow-us-title">Follow Us</h2>
+        <p><small>Join us on social media and stay tuned.</small></p>
+        <p class="network">
+          {% for network in site.data.footer.network %} {% if network.linkedin
+          %}
+          <span>
+            <a href="{{network.linkedin}}" target="_blank">
+              {% include icons/linkedin.svg %}</a
+            ></span
+          >
+          {% endif %} {% if network.twitter %}
+          <span>
+            <a href="{{network.twitter}}" target="_blank">
+              {% include icons/twitter.svg %}</a
+            ></span
+          >
+          {% endif %} {% if network.github %}
+          <span>
+            <a href="{{network.github}}" target="_blank">
+              {% include icons/github.svg %}</a
+            ></span
+          >
+          {% endif %} {% if network.medium %}
+          <span>
+            <a href="{{network.medium}}" target="_blank">
+              {% include icons/medium.svg %}</a
+            ></span
+          >
+          {% endif %} {% if network.meetup %}
+          <span>
+            <a href="{{network.meetup}}" target="_blank">
+              {% include icons/meetup.svg %}</a
+            ></span
+          >
+          {% endif %} {% if network.facebook %}
+          <span>
+            <a href="{{network.facebook}}" target="_blank">
+              {% include icons/facebook.svg %}</a
+            ></span
+          >
+          {% endif %} {% if network.youtube %}
+          <span>
+            <a href="{{network.youtube}}" target="_blank">
+              {% include icons/youtube.svg %}</a
+            ></span
+          >
+          {% endif %} {% if network.instagram %}
+          <span>
+            <a href="{{network.instagram}}" target="_blank">
+              {% include icons/instagram.svg %}</a
+            ></span
+          >
+          {% endif %} {% if network.mail %}
+          <span>
+            <a href="{{network.mail}}" target="_blank">
+              {% include icons/envelope.svg %}</a
+            ></span
+          >
+          {% endif %} {% if network.slack %}
+          <span>
+            <a href="{{network.slack}}" target="_blank">
+              {% include icons/slack.svg %}</a
+            ></span
+          >
+          {% endif %} {% endfor %}
+        </p>
+      </div>
     </div>
+    <div class="row justify-content-between">
+      <div class="col-12 copyright">
+        <p>
+          <small
+            >Women Coding Community is a not-for-profit organisation.</small
```

Comment:
I am not sure which IDEA are you using, but this format is wrong it is breaking the tag in a strange way  `</small` and should always be `</small>` also for ruby code above that happens the same way. Please could you fix that?

Reply:
I reverted this with my latest commit.